web/src/app/admin/embeddings/pages/AdvancedEmbeddingFormPage.tsx

```diff
@@ -131,54 +126,40 @@ const AdvancedEmbeddingFormPage = forwardRef<
                 )}
               </FieldArray>
 
-              <div key="multipass_indexing">
-                <EditingValue
-                  description="Enable multipass indexing for both mini and large chunks."
-                  optional
-                  currentValue={values.multipass_indexing}
-                  onChangeBool={(value: boolean) => {
-                    updateAdvancedEmbeddingDetails("multipass_indexing", value);
-                    setFieldValue("multipass_indexing", value);
-                  }}
-                  setFieldValue={setFieldValue}
-                  type="checkbox"
-                  label="Multipass Indexing"
-                  name="multipassIndexing"
-                />
-              </div>
-              <div key="disable_rerank_for_streaming">
-                <EditingValue
-                  description="Disable reranking for streaming to improve response time."
-                  optional
-                  currentValue={values.disable_rerank_for_streaming}
-                  onChangeBool={(value: boolean) => {
-                    updateAdvancedEmbeddingDetails(
-                      "disable_rerank_for_streaming",
-                      value
-                    );
-                    setFieldValue("disable_rerank_for_streaming", value);
-                  }}
-                  setFieldValue={setFieldValue}
-                  type="checkbox"
-                  label="Disable Rerank for Streaming"
-                  name="disableRerankForStreaming"
-                />
-              </div>
-              <div key="num_rerank">
-                <EditingValue
-                  description="Number of results to rerank"
-                  optional={false}
-                  currentValue={values.num_rerank}
-                  onChangeNumber={(value: number) => {
-                    setRerankingDetails({ ...values, num_rerank: value });
-                    setFieldValue("num_rerank", value);
-                  }}
-                  setFieldValue={setFieldValue}
-                  type="number"
-                  label="Number of Results to Rerank"
-                  name="num_rerank"
-                />
-              </div>
+              <BooleanFormField
```

Comment:
It looks like a lot of this was there before, but for the sake of thoroughness, I'm a little unclear from a user interface perspective if the user is really supposed to understand what these settings do. It feels like a ton of explanation is required.

Reply:
I agree- for BooleanFormField specifically the indent (in this case `noPadding`) is necessary for a few elements in the UI, but I've updated now to be a bit clearer